restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -68,12 +92,22 @@ public void containsIllegalChars() {
 
   @Test
   public void getInsertStatement() {
-    BoundQuery query =
-        documentDB.getInsertStatement(
-            "keyspace", "table", 1, new Object[DocumentDB.allColumns().size()]);
+    Object[] values = new Object[DocumentDB.allColumns().size()];
+    int idx = 0;
+    values[idx++] = "key";
+    for (int i = 0; i < DocumentDB.MAX_DEPTH; i++) {
+      values[idx++] = "value" + i;
+    }
+    values[idx++] = "leaf";
+    values[idx++] = "text";
+    values[idx++] = 1.2d; // dbl_value
+    values[idx++] = true; // bool_value
+    assertThat(idx).isEqualTo(values.length);
```

Comment:
This assertion is weird to me. It seems that we are testing the test init code, is it?
I that it will be easier if we would create a `Map<String, Object>` and use the names as keys:
```
map.put("key", "key");
// ...
map.put("dbl_value", 1.2d);
// ...
```
wdyt?

Reply:
We are asserting that we have values for all all columns (`DocumentDB.allColumns()`). We make some values in a loop and some by hand-written lines, so this assertion is to protect against human errors when editing this code.

`documentDB.getInsertStatement` requires an array of parameters. So I make the array. We can certainly use a map, but to me it seems that the extra changes to handle map in this context may be an overkill.